UmpleToJava/UmpleTLTemplates/JavaClassGenerator.ump

```diff
@@ -389,19 +389,17 @@ for (StateMachine smq : uClass.getStateMachines())
    if (uClass.getAttributes().size()>0 && !matchFound){ #>><<@ UmpleToJava.toString_declare >>
 <<# } #>>
   <<#return realSb.toString();
-    } 
+    }
     private String getAllExtraCode(StringBuilder realSb, UmpleModel model,UmpleClass uClass, GeneratedClass gClass, JavaGenerator gen, boolean isFirst)
   {#>>
-<<# 
+<<#
   boolean isMainClass = false;
-  if (uClass.getExtraCode() != null && uClass.getExtraCode().length() > 0) { #>>  
+  if (uClass.getExtraCode() != null && uClass.getExtraCode().length() > 0) { #>>
   //------------------------
   // DEVELOPER CODE - PROVIDED AS-IS
   //------------------------
-  
-  <<=uClass.getExtraCode()>>
-  
 
+  <<=uClass.getExtraCode()>>
```

Comment:
This change will delete a space that has always been present in such code, rendering many tests failing